service/src/main/resources/application.yml

```diff
@@ -75,7 +75,7 @@ catalog:
     basePath: ${env.rawls.basePath}
 
   schema:
-    basePath: src/main/resources/schema/development/schema.json
+    basePath: classpath:schema/development/schema.json
```

Comment:
Not a blocker to merge, but I'm used to a `basePath` referring to a partial path common to many resources, like how y'all define a `basePath` for interacting with Rawls API above.  This might have a more appropriate name.

Reply:
I'll have Phil look into that on Monday, when he's back, and move forwards with the PR as it stands to unblock monolith release.

Reply:
I agree, the name isn't quite right. I can PR a fix or @Abulu123 could fix it.